### ISSUE TITLE
Why is define_glacier_region needed when running from a copied base_dir?

### ISSUE THREAD

**Issue by anoukvlug** [OGGM/oggm#754]
I was just wondering, why does the `define_glacier_region` gets executed when doing a run for the first time from a `copy_to_basedir (setup='run')` folder? 

It is probably not be worth changing anything about it, as `define_glacier_region` very fast. After reading issue #366 I realised that this issue might get redundant anyway.

**Comment by fmaussion** [comment]
yeah, right now I don't understand anymore what you mean and what I meant in #366 . But yes, I also think some things are a bit clumsy
